src/umpire/resource/HostResourceFactory.cpp

```diff
@@ -46,11 +56,11 @@ HostResourceFactory::create(const std::string& UMPIRE_UNUSED_ARG(name), int id)
   traits.unified = false;
   traits.size = 0;
 
-  traits.vendor = MemoryResourceTraits::vendor_type::IBM;
-  traits.kind = MemoryResourceTraits::memory_type::GDDR;
+  traits.vendor = MemoryResourceTraits::vendor_type::UNKNOWN;
```

Comment:
It would be good to use the vendor detection from #41

Reply:
Agreed. Moving that over!